pkg/nsx/util/utils.go

```diff
@@ -708,3 +708,10 @@ func IsInvalidLicense(err error) bool {
 	}
 	return invalidLicense
 }
+
+func ParseDHCPMode(mode string) string {
+	// Transfer DHCPDeactivated to DHCP_DEACTIVATED
+	nsxMode := strings.ToUpper(mode)
+	nsxMode = nsxMode[:4] + "_" + nsxMode[4:]
```

Comment:
Should add a length check (len(mode) < 5) to prevent slicing errors? BTW, had better a unit-test for this function.